src/urlBuilder.ts

```diff
@@ -72,21 +72,48 @@ export const parseFilters = (
         if (['like', 'ilike'].includes(operation)) {
             // we split the search term in words
             values = filter[key].trim().split(/\s+/);
+        }
+        // CASE: Logical operator
+        else if (['or', 'and'].includes(operation)) {
+            // we extract each value entries and make it dot separated string.
+            const { filter: subFilter } = parseFilters(
+                { filter: filter[key] },
+                defaultListOp
+            );
+
+            const logics: string[] = [];
+            Object.entries(subFilter).forEach(([op, val]) => {
+                if (Array.isArray(val))
+                    logics.push(...val.map(v => [op, v].join('.')));
+                else logics.push([op, val].join('.'));
+            });
+
+            // finally we flatten all as single string and enclose with bracket.
+            values = [`(${logics.join(',')})`];
         } else {
             values = [filter[key]];
         }
 
         values.forEach(value => {
-            // if operator is intentionally blank, rpc syntax
-            let op = operation.includes('like')
-                ? `${operation}.*${value}*`
-                : operation.length == 0
-                ? `${value}`
-                : `${operation}.${value}`;
+            let op: string = (() => {
```

Comment:
In my opinion it would be more convenient to use the type PostgRestOperator as used [here](https://github.com/raphiniert-com/ra-data-postgrest/blob/master/src/urlBuilder.ts#L66)

Reply:
I'm not sure it fits, because `op` is not a single operator anymore (`'eq'`, `'gt'`, `'gte'`, etc...). It will be in the form of: `value`, `{operation}.{value}`, and `{operation}.*{value}*`. Actually I was also not sure to use the variable `op` but I decided to follow the way it was. Maybe `operationQueryValue` name is better IMO